.swift-version

```diff
@@ -1 +1 @@
-3.1.1
+4.1
```

Comment:
This should be `4.0.3` until 4.1 is released, otherwise this would not pass Travis (if we had it turned on).